packages/formstr-app/src/containers/ResponsesNew/index.tsx

```diff
@@ -120,6 +121,16 @@ export const Response = () => {
       [key: string]: string;
     }> = [];
     if (!formSpec || !responses) return;
+    
+    // Check if form has an expiry date
+    const settings = JSON.parse(
+      formSpec.find((tag: any) => tag[0] === "settings")?.[1] || "{}"
+    ) as IFormSettings;
+    
+    let expiryDate: Date | null = null;
+    if (settings.expiryDate) {
```

Comment:
instead of an expiryDate setting, can we use an expiration tag? https://github.com/nostr-protocol/nips/blob/master/40.md

Reply:
not saying we should definitely do this, but what could be the issues?